TimerTriggeredCalc/TimerTriggeredCalc.csproj

```diff
@@ -52,7 +52,9 @@
     <Reference Include="Microsoft.Extensions.Primitives, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Extensions.Primitives.6.0.0\lib\net461\Microsoft.Extensions.Primitives.dll</HintPath>
     </Reference>
-    <Reference Include="OSIsoft.AFSDK, Version=4.0.0.0, Culture=neutral, PublicKeyToken=6238be57836698e6, processorArchitecture=MSIL" />
+    <Reference Include="OSIsoft.AFSDK, Version=4.0.0.0, Culture=neutral, PublicKeyToken=6238be57836698e6, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\Program Files (x86)\PIPC\AF\PublicAssemblies\4.0\OSIsoft.AFSDK.dll</HintPath>
```

Comment:
Is the AFSDK only available in 32-bit?

Reply:
no it's 32 or 64, but there's no public assemblies folder in the %pihome64% branch. What's interesting about this is that I wouldve expected it to pull from the gac_msil, but I guess since that's not a safe path it just references this. I don't really know what the implications of 'hintpath' are.

Reply:
Interesting, from this article it seems like it's just one of many places where it would look https://docs.microsoft.com/en-us/archive/blogs/manishagarwal/resolving-file-references-in-team-build-part-2. 

I am fine with leaving it follows best practices.